Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG BASE_IMAGE=opensuse/leap
+FROM golang:1.12 as build
+ARG user="SUSE CFCIBot"
+ARG email=ci-ci-bot@suse.de
+ADD . /eirini-ssh
+WORKDIR /eirini-ssh
+RUN git config --global user.name ${user}
+RUN git config --global user.email ${email}
+RUN GO111MODULE=on go mod vendor
+RUN bin/build-extension
+RUN bin/build-ssh-proxy
+
+FROM bitnami/kubectl as kubectl
```

Comment:
Would it be possible to grab `kubectl` directly from [upstream] directly? I'm a bit queasy with how difficult it is to track down the source of the bitnami images.

We have a [kubernetes-client] package in OBS, but that's not built with `CGO_ENABLED=0` so it is (probably) dynamically linked and requires using the appropriate base image.

[upstream]: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md#client-binaries
[kubernetes-client]: https://software.opensuse.org/package/kubernetes-client